noir-projects/noir-contracts/contracts/auth_contract/src/test/main.nr

```diff
@@ -0,0 +1,119 @@
+use crate::test::utils;
+use crate::Auth;
+
+use dep::aztec::prelude::AztecAddress;
+
+#[test]
+unconstrained fn main() {
```

Comment:
Let's add a todo mentioning this was simply ported from the e2e flow - at some point we'd want to rewrite this in TXE-style and e.g. check the error strings.